src/webview/html/editHistoryHtml.js

```diff
@@ -0,0 +1,105 @@
+/* @flow strict-local */
+import template from './template';
+import type { ThemeName, MessageSnapshot, UserOrBot, Auth } from '../../types';
+import { shortTime, humanDate } from '../../utils/date';
+import css from '../css/css';
+import script from '../js/script';
+
+const TOPIC_EDITED = 'TOPIC_EDITED';
+const CONTENT_EDITED = 'CONTENT_EDITED';
+const BOTH_EDITED = 'BOTH_EDITED';
+const NOT_EDITED = 'NOT_EDITED';
+
+const renderSnapshot = (snapshot: MessageSnapshot, usersById: Map<number, UserOrBot>): string => {
+  const user = usersById.get(snapshot.user_id);
+  const date = new Date(snapshot.timestamp * 1000);
+  const editedTime = `${humanDate(date)} ${shortTime(date)}`;
+  let displayedContent: string = '';
+  let tag: string = '';
+  let edit_flag;
+
+  if (snapshot.prev_topic === undefined && snapshot.prev_content === undefined) {
+    edit_flag = NOT_EDITED;
+  } else if (snapshot.prev_content !== undefined && snapshot.prev_topic === undefined) {
+    edit_flag = CONTENT_EDITED;
+  } else if (snapshot.prev_content === undefined && snapshot.prev_topic !== undefined) {
+    edit_flag = TOPIC_EDITED;
+  } else {
+    edit_flag = BOTH_EDITED;
+  }
+
+  switch (edit_flag) {
+    case NOT_EDITED: {
+      tag += template`<span class="message-tag">Original message</span>`;
```

Comment:
When you're looking at the original message, is the Markdown rendered properly?

I'm seeing the raw text; not sure why. But it seems like it's been possible to show everything properly formatted when viewing edits in the list, so it would be great to get the original message formatted properly too if we can (and it's that way in the web app). 🙂

Reply:
Oops, I placed `content` instead of `rendered_content`!